### PR TITLE
update readme with sync instructions

### PR DESCRIPTION
**Pull request by uditvira**
* adds instructions on state-sync
* adds an example script to sync a full node quickly

### CODE REVIEW

**Review by yaruwangway — APPROVED** [review]

File: public-testnet/README.md [hunk 1]
```diff
@@ -57,3 +65,172 @@ To use Cosmovisor to manage your upgrade, please follow the [Cosmovisor instruct
 Make sure your machine is resourced with 16GB while performing the upgrade. The upgrade process is memory intensive.
 
 **Note about auto-downloads:** If validators would like to enable the auto-download option (which we don't recommend), and they are currently running an application using Cosmos SDK v0.42, they will need to use Cosmovisor v0.1. Later versions of Cosmovisor do not support Cosmos SDK v0.42 or earlier if the auto-download option is enabled. Please note that with v0.1 you could face node hanging issues with your API server enabled as explained in this [issue](https://github.com/cosmos/cosmos-sdk/issues/9875). If your node is running on darwin, arm64 CPU architecture, please do not use cosmovisor auto-download because the upgrade proposal will not provide the binary download link for this architecture. Please prepare the new binary yourself from this [v6.0.0-rc3 tag](https://github.com/cosmos/gaia/tree/v6.0.0-rc3).
+
+## Using state-sync
+
+We're serving snapshots every 1000 blocks from the following three nodes. Their p2p listen addresses are (`<node-id>@<public-ip>:<port>`):
+
+* `5303f0b47c98727cd7b19965c73b39ce115d3958@134.122.35.247:26656`
```
[the remark this answers is not in the document]
Comment (uditvira) [reply public-testnet/README.md:73]: yes

File: public-testnet/README.md [hunk 2]
```diff
@@ -57,3 +65,172 @@ To use Cosmovisor to manage your upgrade, please follow the [Cosmovisor instruct
...
+* `9e1e3ce30f22083f04ea157e287d338cf20482cf@165.22.235.50:26656`
+* `b7feb9619bef083e3a3e86925824f023c252745b@143.198.41.219:26656`
+
+Add these to your persistent_peers list to help your nodes discover snapshots quickly. To enable snapshot discovery, you'll need to configure the `[statesync]` section of your `config.toml` file. You'll need to set `enable = true`, set a `trust_height`, a corresponding `trust_hash` (easily found on a block explorer), and atleast two trusted RPC servers that your node will use to cross-check hashes. Note than in the future, the RPC server requirement will be deprecated as state sync is [moved to the p2p layer in Tendermint 0.35](https://github.com/tendermint/tendermint/issues/6491).
```
Comment (yaruwangway) [inline public-testnet/README.md:77]: what would be a good `trust_height` and for `trust_hash` is just any hash of the past blocks?

Comment (yaruwangway) [reply public-testnet/README.md:77]: `trust_height` and for `trust_hash ` have to be from the same block ?

Comment (uditvira) [reply public-testnet/README.md:77]: yes same block for `trust_height` and `trust_hash`. I think ideally a block height that is lower than the last snapshot that was taken

Comment (yaruwangway) [reply public-testnet/README.md:77]: atleast -> at least

**Pull request merged**